README.md

```diff
@@ -44,176 +49,341 @@ try {
 }
 ```
 
-### Commands
+### API
 
-Anywhere a promise is returned from the bot API, you should handle error with `.cathc(err => { ... })`
+<!-- Generated by documentation.js. Update this documentation by updating the source code. -->
 
-#### `bot.init(options)`
+##### Table of Contents
```

Comment:
Is there any way to bump up the headings on the ToC? It's a little weird to go from h1 and 2s to h5s and 6s once we reach this point

Reply:
That's done automatically by the script. As best I can tell I am not able to control it. I can tweak the parameters around `documentation.yml` to see if I can reduce the header level.

Reply:
@nathunsmitty The answer is that the table of contents and the Markdown that is output from the library is two levels deeper than the section it is mounted to (e.g. `###API` → `##### Table of Contents`)

The markdown output cannot be customized but the HTML template out can be and included in the README if necessary.